src/adsb_actions/adsbactions.py

```diff
@@ -21,16 +20,22 @@
 from io import StringIO
 from typing import Callable
 
-from .rules import Rules
-from .flights import Flights
-from .bboxes import Bboxes
-from .stats import Stats
-from .location import Location
+from rules import Rules
+from flights import Flights
+from bboxes import Bboxes
+from stats import Stats
+from location import Location
 
 # Prometheus exporter
 from prometheus_client import start_http_server, Gauge
 
-logger = logging.getLogger(__name__)
+import adsb_logger
+from adsb_logger import Logger
+ 
+logger = adsb_logger.logging.getLogger(__name__)
+#logger.level = adsb_logger.logging.DEBUG
+LOGGER = Logger()
```

Comment:
You changed some lines below to refer to "logger" instead of "LOGGER".  Is the intention that we switch everything over to LOGGER in a subsequent change?

Reply:
Oh, nevermind, I see what's happening now, this line is here just to cause the __init__ to change the logger parameters.  I wonder if that means that adsb_logger should inherit the logging module, but that's a bit beyond my python expertise...

Reply:
Yep, just for init.  :^)  Hmm, adsb_logger imports the logging module.  Sheesh!  Module shmodule, I'm gaining on it but don't know yet.